### DISCUSSION TITLE
trying to build on windows and problems with svg when building gi

### DISCUSSION THREAD

**Discussion by jkyle1234** [cogentcore/core#436] [Q&A] [answered]
In the examples/widgets folder I am running go get./.. but I keep hitting the issue below 

Trying to build gi on windows but I keep running into issues with svg not finding the references to the mat32 matrix class. Any idea how to fix this?

(base) C:\Go Projects\gi\examples\widgets>go get ./...
# github.com/goki/gi/svg
..\..\svg\circle.go:71:12: xf.MulVec2AsPtCtr undefined (type mat32.Mat2 has no field or method MulVec2AsPtCtr)
..\..\svg\ellipse.go:70:12: xf.MulVec2AsPtCtr undefined (type mat32.Mat2 has no field or method MulVec2AsPtCtr)
..\..\svg\image.go:163:12: xf.MulVec2AsPtCtr undefined (type mat32.Mat2 has no field or method MulVec2AsPtCtr)
..\..\svg\line.go:79:14: xf.MulVec2AsPtCtr undefined (type mat32.Mat2 has no field or method MulVec2AsPtCtr)
..\..\svg\line.go:80:12: xf.MulVec2AsPtCtr undefined (type mat32.Mat2 has no field or method MulVec2AsPtCtr)
..\..\svg\node.go:116:20: g.MyXForm().Inverse undefined (type mat32.Mat2 has no field or method Inverse)
..\..\svg\path.go:884:10: xf.MulVec2AsPtCtr undefined (type mat32.Mat2 has no field or method MulVec2AsPtCtr)
..\..\svg\path.go:946:13: xf.MulVec2AsPtCtr undefined (type mat32.Mat2 has no field or method MulVec2AsPtCtr)
..\..\svg\path.go:960:13: xf.MulVec2AsPtCtr undefined (type mat32.Mat2 has no field or method MulVec2AsPtCtr)
..\..\svg\polygon.go:95:9: xf.MulVec2AsPtCtr undefined (type mat32.Mat2 has no field or method MulVec2AsPtCtr)
..\..\svg\polygon.go:95:9: too many errors

**Comment by rcoreilly** [comment, marked as answer]
sorry should be fixed now -- updated mat32 version to 1.0.3

**Reply by jkyle1234** [reply]
Thanks Dr. O'Reilly that worked now. I was having an issue with the gcc compiler after that, but I found if you add the -buildmode exe onto the build command it works for windows. I'm going through your excellent course Computational Cognitive Neuroscience on youtube now and I think leabra is brilliant.

c:\go\pkg\tool\windows_amd64\link.exe: running gcc failed: exit status 1
C:/TDM-GCC-64/bin/../lib/gcc/x86_64-w64-mingw32/9.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: error: export ordinal too large: 86592
collect2.exe: error: ld returned 1 exit status


(base) C:\Go Projects\leabra\examples\ra25>go build -buildmode exe

**Comment by rcoreilly** [comment]
Thanks!  finally added the -buildmode exe to Install instructions -- not sure if it is fixed or not yet.. https://github.com/golang/go/issues/40795 seems a bit ambiguous
